client/package-lock.json

```diff
@@ -1,8804 +0,0 @@
-{
```

Comment:
please keep this file. I added it recently to fix an issue with building Docker image. 
It's not easy to integrate existing yarn setup with how we build our images and in the same time, we don't want to do fresh install via npm during Docker build step. (i faced with multiple issues related to unexpected package updates which led to downtime:))